SynchronizationClock.md

```diff
@@ -14,15 +14,14 @@ The `Harp Synchronization Clock` is a dedicated bus that disseminates the curren
 
 * The packet is composed of 6 bytes (`header[2]` and `timestamp_s[4]`):
   - `header[2] = {0xAA, 0xAF)`
-  - `timestamp_s` is of type U32, little-endian, and contains the current second.
+  - `timestamp_s` is of type U32, little-endian, and contains the previous elapsed second.
 
-> **Important**
->
-> To avoid unexpected behaviors, only one bit at a time should be written to register `R_RESET_DEV`.
+A sample logic trace is shown below:
+    !["SynchClockLogicAnalyzer](./assets/synch_logic_trace.png)
```

Comment:
```suggestion
    ![SynchClockLogicTrace](./assets/SynchClockLogicTrace.png)
```

Similar to the other assets, can we rename this file to follow PascalCasing? In this case I propose a mix keeping the alt tag casing and the "trace" word for clarity.

Reply:
Done!